vscode-typescript/track.yml

```diff
@@ -0,0 +1,50 @@
+slug: vscode-typescript
+id: 0dxngfkws4op
+type: track
+title: VSCode / TypeScript
+teaser: Edit and test TypeScript applications in VSCode
+description: |-
+  A basic track for editing a TypeScript application and running tests for it.
+  The track checks are failing based on the test results.
+
+  ## Using this template
+  To create your own track from this template, run this command:
+  ```
+  instruqt track create \
+      --from templates/container \
+      --title "My VSCode / TypeScript track"
+  ```
+icon: https://cdn.jsdelivr.net/gh/remojansen/logo.ts@1ff5efb/ts.png
```

Comment:
I have uploaded this image to our CDN: https://cdn.instruqt.com/assets/templates/ts.png

Reply:
Updated: https://github.com/instruqt/templates/pull/3/commits/a3ae087085fc272048cfcbc985c12d8b719b9513.